src/test/java/seedu/doctorappointments/DoctorAppointmentCheckerTest.java

```diff
@@ -0,0 +1,62 @@
+package seedu.doctorappointments;
+
+import org.junit.jupiter.api.Test;
+import seedu.model.doctorappointment.AppointmentList;
+import seedu.logic.errorchecker.DoctorAppointmentChecker;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DoctorAppointmentCheckerTest {
+
+    @Test
+    public void testValidAppointmentId() {
```

Comment:
Perhaps you can consider a testInvalidAppointmentID as well to increase code coverage.